fast/stages/1-resman/stage-3.tf

```diff
@@ -108,6 +111,22 @@ locals {
       ]
     ]
   ])
+  stage3_factories = [
+    for k, v in local.stage3 :
+    v.factories_config
+    if v.factories_config != null
+  ]
+}
+
+# Stage 3 custom role factories
+
+module "organization-roles" {
```

Comment:
do we really need extra module calls for this? can't we roll up modules in the existing organization module?